airflow/providers/google/suite/hooks/calendar.py

```diff
@@ -0,0 +1,213 @@
+
+    def get_conn(self) -> Any:
+        """
+        Retrieves connection to Google Calendar.
+
+        :return: Google Calendar services object.
+        :rtype: Any
+        """
+        if not self._conn:
+            http_authorized = self._authorize()
+            self._conn = build('calendar', self.api_version, http=http_authorized, cache_discovery=False)
+
+        return self._conn
+
+    def get_events(
+        self,
+        calendar_id: str = 'primary',
+        i_cal_uid: Optional[str] = None,
+        max_attendees: Optional[int] = None,
+        max_results: Optional[int] = None,
+        order_by: Optional[str] = None,
+        private_extended_property: Optional[str] = None,
+        q: Optional[str] = None,
+        shared_extended_property: Optional[str] = None,
+        show_deleted: Optional[bool] = False,
+        show_hidden_invitation: Optional[bool] = False,
+        single_events: Optional[bool] = False,
+        sync_token: Optional[str] = None,
+        time_max: Optional[datetime] = None,
+        time_min: Optional[datetime] = None,
+        time_zone: Optional[str] = None,
+        updated_min: Optional[datetime] = None,
+    ) -> list:
+        """
+        Gets events from Google Calendar from a single calendar_id
+        https://developers.google.com/calendar/api/v3/reference/events/list
+
+        :param calendar_id: The Google Calendar ID to interact with
+        :param i_cal_uid: Optional. Specifies event ID in the iCalendar format in the response.
+        :param max_attendees: Optional. If there are more than the specified number of attendees,
+            only the participant is returned.
+        :param max_results: Optional. Maximum number of events returned on one result page.
+            Incomplete pages can be detected by a non-empty nextPageToken field in the response.
+            By default the value is 250 events. The page size can never be larger than 2500 events
+        :param order_by: Optional. Acceptable values are "startTime" or "updated"
+        :param page_token: Optional. Specifies result page to be returned.
+        :param private_extended_property: Optional
+        :param q: Optional. Free text search.
+        :param shared_extended_property: Optional
+        :param show_deleted: Optional. False by default
+        :param show_hidden_invitation: Optional. False by default
+        :param single_events: Optional. False by default
+        :param sync_token: Optional. Token obtained from the nextSyncToken field returned
+        :param time_max: Optional. Upper bound (exclusive) for an event's start time to filter by.
+            Default is no filter
+        :param time_min: Optional. Lower bound (exclusive) for an event's end time to filter by.
+            Default is no filter
+        :param time_zone: Optional. Time zone used in response. Default is calendars time zone.
+        :param updated_min: Optional. Lower bound for an event's last modification time
```

Comment:
Can the param types be added to the docstring? These docstrings are used to build API documentation for the hook and includes the name, description, and expected arg type for each parameter. For example, the [docs](https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/suite/hooks/sheets/index.html) for the `GSheetsHook`.

Reply:
Updated